src/Dfe.PlanTech.Web/Views/Cookies/CookiesPreferencesSetBanner.cshtml

```diff
@@ -0,0 +1,26 @@
+@using Dfe.PlanTech.Domain.Cookie;
+
+@{
+    bool userPreferenceSet = TempData[CookieConstants.UserPreferenceRecordedKey] as bool? ?? false;
+}
+
+@if (userPreferenceSet)
+{
+    <div class="govuk-notification-banner govuk-notification-banner--success" role="alert"
+        aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                Success
+            </h2>
+        </div>
+        <div class="govuk-notification-banner__content">
+            <p class="govuk-notification-banner__heading">
+                You’ve set your cookie preferences.
+                
+                <a class="govuk-notification-banner__link"
+                    href="javascript:void(0)"
+                    onclick="window.history.go(-2)">Go back to the page you were looking at</a>.
```

Comment:
This isn't gonna work with csp no unsafe inline, I think you can't hash it either so may need to do it with listener

Reply:
I didn't think of that (this is from the original version). I'll amend the process so it doesn't use Javascript.

Reply:
Amended. Can you try break it please? 👼

Reply:
I have tried, and I think been successful! If I go from self assessment page to cookies and accept, it redirects me to the cookies page